ios/Rainbow.xcodeproj/project.pbxproj

```diff
@@ -758,7 +758,7 @@
 					13B07F861A680F5B00A75B9A = {
```

Comment:
Not sure how all these things got changed, should I remove this file from the PR?

Reply:
yes